app/views/organizations/organization_profiles/_form.html.erb

```diff
@@ -0,0 +1,76 @@
+<%= form_with model: profile, :url => organization_profile_path do |form| %>
```

Comment:
We've just merged the bootstrap_form gem, which simplifies form creation significantly by auto-handling labels and validations 🚀 Could you update this form to utilize it?
Thanks!

Reply:
Btw, we can also do it in a separate PR and merge this one faster :)